lib/bolt/catalog/loaders.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Puppet
```

Comment:
Can we make the file match the class path by moving the class or the file?

Reply:
This is following the pattern you established in catalog/compiler.rb

Reply:
So not sure what you're asking me to do.

Reply:
I expect the compiler class to go away when we switch to PAL. It's in Puppet because it was a pain to namespace stuff otherwise.
If this would require excesssive namespacing it should probably go int puppet/pops/bolt_loaders otherwise we can just move it to Bolt::Catalog::Loaders

Reply:
I'll try it out with Bolt::Catalog::Loaders.

Reply:
`Bolt::Catalog` is a class, so it looks a little funny, but it works.

Reply:
Updated.